content/projects/gsoc/students.adoc

```diff
@@ -50,31 +81,203 @@ We highly recommend to make some contributions to the project while you work on
 It will help you to polish the proposal,
 and mentors will consider contributions and interactions with the community when processing applications.
 
-Useful links:
+[[UsefulLinks]]
+=== Useful links
+
+Here are a list of links to help you get started on participating in Jenkins and in coding for the Jenkins project,
+in increasing level of difficulty.
+
+* link:https://jenkins.io/project/governance/[Project Governance]
+* link:https://github.com/jenkinsci/jenkins/blob/master/CONTRIBUTING.md[Contributing to Jenkins] (for contributions to the core)
+* link:https://wiki.jenkins.io/display/JENKINS/Beginners+Guide+to+Contributing[Beginners Guide to Contributing]
+* link:https://wiki.jenkins.io/display/JENKINS/Plugin+tutorial[Plugin tutorial]
+* link:https://jenkins.io/doc/developer/tutorial/[Another plugin tutorial]
+* link:https://jenkins.io/blog/2017/08/07/intro-to-plugin-development/[Plugin Development Tutorials, Videos, and More]
+* link:https://jenkins.io/doc/developer/plugin-development/pipeline-integration/[Writing Pipeline-Compatible Plugins]
+* link:https://jenkins.io/doc/developer/extensions/[Extension Points]
+* link:https://wiki.jenkins.io/display/JENKINS/Hosting+Plugins[Releasing plugins]
```

Comment:
Not needed for beginners

Reply:
Moved under an advanced title.

Reply:
Maybe linking to the developer guidelines would be enough. I do not want to overload the page